include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -2556,6 +2556,17 @@ class FEEvaluation : public FEEvaluationAccess<dim,
     const unsigned int active_fe_index   = numbers::invalid_unsigned_int,
     const unsigned int active_quad_index = numbers::invalid_unsigned_int);
 
+  /**
+   * Constructor. Takes all data stored in MatrixFree for a given cell range,
+   * which allows to automatically identify the active_fe_index and
+   * active_quad_index in case of a p-adaptive strategy.
```

Comment:
Would you mind also adding that the rest of the arguments are the same as in the constructor above.

Reply:
Done! I have rebased this branch!